frontend/src/theme/components/Filter.ts

```diff
@@ -0,0 +1,12 @@
+const Filter = {
+  baseStyle: {
+    backgroundColor: "#FAFAFA",
```

Comment:
see comment about colors being in `frontend/src/theme/colors.ts`? `#FAFAFA` is frequently used color

edit: OH FANCY IT SHOWS A COLOUR ICON